packages/common/src/groups/_internal/computeProps.ts

```diff
@@ -5,6 +5,9 @@ import { IHubGroup } from "../../core/types/IHubGroup";
 import { IGroup } from "@esri/arcgis-rest-types";
 import { isDiscussable } from "../../discussions";
 import { getGroupThumbnailUrl } from "../../search/utils";
+import { getRelativeWorkspaceUrl } from "../../core";
+import { getGroupHomeUrl } from "../../urls";
```

Comment:
I don't see these being used in this file, might have been left here by accident?